src/collections/_documentation/development/sdk-dev/interfaces/gpu.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'GPU Interface'
+sidebar_order: 13
+---
+
+An interface which describes the main GPU of the device.
+
+`name`
```

Comment:
@jessla brought up that it'd be ideal if we could make these headings so they're linkable. Any reason to not do that?

Reply:
@cameronmcefee I think it would be great if we could link to individual headings.
Something to be done in a separate PR I assume though?

Reply:
I think it'd be fine to do it here. All it takes is switching these to headings. If you want to retain the code highlight, you could do 

```
### `name`
```

Reply:
Oh, I thought you were referring to adding to all the `interfaces` docs since they are also like this.
I'll add the headings here as you suggested.